### ISSUE TITLE
Can't upload a .psd

### ISSUE THREAD

**Issue by olixem** [avocode/avocode#943]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.8.1
Source UID: 72431
```
TypeError: Cannot read property 'get' of null
    at Function.e.fromProject (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:14:13506)
    at e.createUploadTargetFromProject (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:14:1668)
    at Constructor._handleUploadNewDesign (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:34:21429)
    at Constructor._handleUploadNewDesign (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:36:7145)
    at Object.invokeGuardedCallback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactErrorUtils.js:27:12)
    at executeDispatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/EventPluginUtils.js:79:21)
    at Object.executeDispatchesInOrder (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/EventPluginUtils.js:99:7)
    at executeDispatchesAndRelease (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/EventPluginHub.js:43:22)
    at executeDispatchesAndReleaseTopLevel (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/EventPluginHub.js:54:10)
    at Array.forEach (native)
    at forEachAccumulated (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/forEachAccumulated.js:23:9)
    at Object.processEventQueue (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/EventPluginHub.js:259:7)
    at runEventQueueInBatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactEventEmitterMixin.js:18:18)
    at Object.handleTopLevel [as _handleTopLevel] (/Applications/Avocode.app/Contents/Resources/app.asar
```

**Comment by jakubzitny** [comment]
Hi @olixem, thanks for the report. We already know about this and we will release new update for Avocode today with fix for it.

**Comment by jakubzitny** [comment]
We released 2.8.2 today, so I will close this issue. If it occurs again or you have any other problems with Avocode, feel free to let us know ;)

New Avocode update is available directly in the app or at [avocode.com/download](http://avocode.com/download/).
